flake.nix

```diff
@@ -34,7 +34,7 @@
 
     location-tracking-service.url = "github:nammayatri/location-tracking-service";
     passetto = {
-      url = "github:nammayatri/passetto/nixify";
+      url = "github:nammayatri/passetto/use-crypton";
```

Comment:
Can you link to the PR in the code comment here?

Reply:
Done

Reply:
Forgot to push?